ITEventsUITests/FullEventViewUITests.swift

```diff
@@ -0,0 +1,114 @@
+import XCTest
+
+class FullEventViewUITests: XCTestCase {
+    private var app: XCUIApplication!
+    
+    private lazy var eventView = app.otherElements.matching(identifier: "EventView")
+    private lazy var eventImage = app.images.matching(identifier: "EventImage")
+    private lazy var eventTitle = app.staticTexts.matching(identifier: "EventTitle")
+    private lazy var eventDate = app.staticTexts.matching(identifier: "EventDate")
+    private lazy var eventCity = app.staticTexts.matching(identifier: "EventCity")
+    private lazy var eventAddress = app.staticTexts.matching(identifier: "EventAddress")
+    private lazy var eventDescription = app.staticTexts.matching(identifier: "EventDescription")
+    private lazy var eventTags = app.otherElements.matching(identifier: "EventTags")
+    private lazy var eventSourceButton = app.buttons.matching(identifier: "EventSourceButton")
+    private lazy var showSimilarEventsButton = app.otherElements["ShowSimilarEventsButton"]
```

Comment:
what is difference between these approaches?
```swift    
private lazy var button1 = app.buttons.matching(identifier: "ShowSimilarEventsButton")
private lazy var button2 = app.otherElements["ShowSimilarEventsButton"]
```

Reply:
First returns ElementQuery, second - Element. So 
```app.buttons.matching(identifier: "ShowSimilarEventsButton").element == app.otherElements["ShowSimilarEventsButton"]```
In almost all cases I need ElementQuery but for ```showSimilarEventsButton``` Element is enough